recipe/build-pyqt.sh

```diff
@@ -43,6 +43,8 @@ $SIP_COMMAND \
 --verbose \
 --confirm-license \
 --no-make \
+--qt-shared \
```

Comment:
Does this disable `qtwebkit`? It is not clear how it is being disabled?

Reply:
`pyqtwebkit` is built if `qtwebkit` is available, otherwise not. The same build script is used for both `pyqt` and `pyqtwebkit`; the difference is that the `host` requirements for `pyqt` does not include `qtwebkit` (and so `pyqtwebkit` is not built there) while the `host requirements for `pyqtwebkit` do include `qtwebkit`, so `pyqtwebkit` is built there.

Reply:
Does the `pyqtwebkit` build overwrites files that were generated by the `pyqt` build? It seems a little bit opaque, isn't it possible to build `pyqtwebkit` explicitly?

Reply:
No it doesn't overwrite anything. I've uploaded the packages to my `izahn` channel, you can inspect them there if you like.

Reply:
`pyqtwebkit` isn't a separate thing from `pyqt`, so no it doesn't make sense to build it explicitly. conda build does pull them apart for us, which is very nice.

Reply:
Fedora does it the same way, everything is build together and then pulled apart: https://src.fedoraproject.org/rpms/python-qt5/blob/rawhide/f/python-qt5.spec

They just don't have `conda-build` to do it for them automagically :-)

Reply:
Easy there. conda-build is not even close to what a good specfile and rpbuild can achieve.

I am surprised that Fedora still ships QtWebKit.

Reply:
Debian still ships QtWebKit too, and their QGIS also depends on it: https://packages.debian.org/bullseye-backports/python3-qgis

Reply:
@andfoy I enabled azure artifact storage so you can inspect all packages produced by this build at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=546950&view=artifacts&pathAsName=false&type=publishedArtifacts . I've reviewed them and they all look good to me. If you agree I'll mark this conversation as resolved.